UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartDamage.cs

```diff
@@ -183,6 +183,12 @@ public float TotalDamage
 			}
 		}
 
+		/// <summary>
+		/// So functions can subscribe and do there Funky stuff when damage is taken
+		/// </summary>
+		public event Action<AttackType,DamageType, float> OnDamageChangeBy;
```

Comment:
I can't believe we didn't have this event already. I have some suggestions tho.
```suggestion
		public event Action<AttackType,DamageType, float> OnDamageTaken;
```
Current name suggests you can get who did the damage or so I understand.

Reply:
it describes the change in damage?

Reply:
maybe change isn't the right word, uhmm